ansible/platform_skip_inventory.json

```diff
@@ -0,0 +1,9 @@
+{
+   "dut2": {
+       "modules": {
+                "skip": ["FABRIC-CARD0", "LINE-CARD4"]
+           }
+      }
```

Comment:
If you want to determine which modules to test or skip on a DuT basis, it might be better to add attributes to the inventory file rather than adding a new JSON file.